packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -12,14 +12,31 @@ import 'tabs.dart';
 import 'theme.dart';
 import 'typography.dart';
 
+/// A material design app bar.
+///
+/// An app bar consists of a tool bar and potentially other widgets, such as a
+/// tab strip and flexible space. App bars typically have a number of actions,
```

Comment:
Suggestion:
App bars typically expose one or more common actions with [IconButtons]s which are optionally followed by a [PopupMenuButton] for less common operations.